src/content/blog/astro-tsc-2025.mdx

```diff
@@ -0,0 +1,40 @@
+---
+title: '2025 Technical Steering Committee'
+description: 'Meet Astro’s 2025 Technical Steering Committee and learn how our open governance model supports the framework’s growth, performance, and community-led development.'
+publishDate: '2025-04-18'
+authors:
+  - fred
+coverImage: '/src/content/blog/_images/astro-tsc-2025/blog-hero.jpg'
+socialImage: '/src/content/blog/_images/astro-tsc-2025/og-image.jpg'
+lang: 'en'
+---
+
+import Mention from "/src/components/Mention.astro"
+
+The Astro project has a new Technical Steering Committee (TSC) for 2025! I am excited to announce <Mention name="ema" /> <Mention name="florian" /> and <Mention name="matt" /> are now part of the Astro TSC, joining Matthew Phillips who will remain on the TSC. 
```

Comment:
Just pointing out that Matthew is the only one here not `<Mentioned>`, in case that wasn't intentional!

Reply:
whoops! Thanks!